src/LessMsi.Cli/eExtractionMode.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace LessMsi.Cli
+{
+    public enum eExtractionMode
```

Comment:
Can you please rename this to ExtractionMode per https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations ?

Reply:
Will amend the code accordingly.